AFQ/tests/test_api.py

```diff
@@ -349,7 +349,7 @@ def test_AFQ_data_waypoint():
     tgram = load_tractogram(myafq.bundles[0], myafq.dwi_img[0])
 
     bundles = aus.tgram_to_bundles(tgram, myafq.bundle_dict, myafq.dwi_img[0])
-    npt.assert_(len(bundles['CST_R']) > 0)
+    npt.assert_(len(bundles['CST_L']) > 0)
```

Comment:
Do you understand why this changed? That is, which of the other changes in this PR required that this be changed?

Reply:
I will say that I took a look at the documentation example, and the segmentation, as well as tract profile does look god, but I am always a bit worried when the segmentation changes, even if it's for this rather skeletal test case.

Reply:
This only uses 100 streamlines to be fast. I think the update to dipy might have changed it? I could up the sls to 200 to see if that finds the CST_R.

Reply:
OK - if it's the DIPY upgrade, then that's probably fine.

Reply:
Actually, I think its because I wasn't using the subsampled tractography anymore because it was saved with the uppercase model-DTI.
Due to changes recommended in another comment, everything uses uppercase again. But I am going to remove the subsampled tractography because I want to use the ROIMask for tractography instead.

Reply:
Oh good! This one test is one that has proven to be a really useful canary in the coal-mine in the past, so whenever it fails, it raises questions. But I think that changing as you suggest should be fine.